src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/CreateCaseCallbackTest.java

```diff
@@ -27,9 +28,7 @@
 @IntegrationTest
 class CreateCaseCallbackTest {
 
-    private static final String CASE_ID_REGEX =
-        // will be replaced with real one after ccd api
-        "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$";
+    private static final String CASE_ID_REGEX = "\\d+";
```

Comment:
Wy don't we verify that values are exactly the same?

Reply:
Let me update this then. Was quickly renaming it but intension was to actually copy from json files. I'll do that and add a comment